docker/Dockerfile

```diff
@@ -1,63 +1,44 @@
-FROM  python:3.9-alpine@sha256:e80214a705236091ee3821a7e512e80bd3337b50a95392a36b9a40b8fc0ea183  as  build
+FROM alpine/git as git-clone
 
-LABEL "maintainer"="FIWARE Foundation e.V."
-LABEL "description"="An Internet of Things Agent for the RDF DataQube Turtle format. This IoT Agent is designed to be a bridge between ISOXML/ADAPT and the ETSI NGSI-LD interface of a FIWARE Context Broker."
-LABEL "name"="iotagent-turtle"
-LABEL "summary"="IoT Agent for the DataQube representation in RDF Turtle format used in statistical environments"
+ARG PROJECT=flopezag
+ARG COMPONENT=IoTAgent-Turtle
+ARG VERSION=develop
+ARG INSTALLATION_PATH=/opt/$COMPONENT
 
-LABEL "org.opencontainers.image.authors"="fernando.lopez@fiware.org"
-LABEL "org.opencontainers.image.documentation"=""
-LABEL "org.opencontainers.image.vendor"="FIWARE Foundation e.V."
-LABEL "org.opencontainers.image.licenses"="Apache2.0"
-LABEL "org.opencontainers.image.title"="iotagent-turtle"
-LABEL "org.opencontainers.image.description"="An Internet of Things Agent for the RDF DataQube Turtle format. This IoT Agent is designed to be a bridge between ISOXML/ADAPT and the ETSI NGSI-LD interface of a FIWARE Context Broker."
-LABEL "org.opencontainers.image.source"="https://github.com/flopezag/IoTAgent-Turtle"
-LABEL "org.opencontainers.image.version"="0.1.0"
-LABEL "org.python.version"="python:3.9"
+RUN mkdir -p $INSTALLATION_PATH
+RUN git clone https://github.com/$PROJECT/$COMPONENT --branch $VERSION $INSTALLATION_PATH
 
-RUN  apk  update
-# RUN apt-get install -y --no-install-recommends \
-# 	build-essential gcc
 
-WORKDIR  /usr/app
-RUN  python  -m  venv  /usr/app/venv
-ENV  PATH="/usr/app/venv/bin:$PATH"
+## Install PIP Requirements
+FROM python:3.11-alpine as pip-requirements
 
-COPY  requirements.txt  .
-RUN  pip  install  --no-cache-dir  --upgrade  -r  requirements.txt
+ARG PROJECT=flopezag
+ARG COMPONENT=IoTAgent-Turtle
+ARG VERSION=integration-cb-httpserver
+ARG INSTALLATION_PATH=/opt/$COMPONENT
 
-FROM python:3.9-alpine@sha256:e80214a705236091ee3821a7e512e80bd3337b50a95392a36b9a40b8fc0ea183
+RUN mkdir -p $INSTALLATION_PATH
+COPY --from=git-clone $INSTALLATION_PATH/requirements.txt /requirements.txt
+RUN pip install --root-user-action=ignore --prefix=$INSTALLATION_PATH -r /requirements.txt
 
-ENV PORT=${IOTA_PORT:-5000}
-
-LABEL "maintainer"="FIWARE Foundation e.V."
```

Comment:
why did you delete this information, at least in the last layer we should provide this information but by checking the content we should modify some of them:

LABEL "description"="An Internet of Things Agent for the RDF Turtle of SDMX DataQube format. This IoT Agent is designed to be a bridge between SDMX statistical metadata representation in RDF Turtle and  ETSI NGSI-LD in JSON-LD format representation to be integrated with FIWARE components."

LABEL "summary"="IoT Agent for the DataQube (SDMX) representation in RDF Turtle format used in statistical environments."

LABEL "org.opencontainers.image.documentation"="https://github.com/flopezag/IoTAgent-Turtle/tree/master/doc"

LABEL "org.opencontainers.image.description"="An Internet of Things Agent for the RDF Turtle of SDMX DataQube format. This IoT Agent is designed to be a bridge between SDMX statistical metadata representation in RDF Turtle and  ETSI NGSI-LD in JSON-LD format representation to be integrated with FIWARE components."

LABEL "org.python.version"="python:3.11"

Reply:
Reset

Reply:
Please check the comment because I have changed some of the values.

Reply:
OK